pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php

```diff
@@ -335,7 +335,7 @@ public function additional_meta_box_fields( $order_id, $is_disabled, $dhl_label_
 						)
 					);
 
-					$this->crossborder_and_domestic_fields( $dhl_label_items, $is_disabled );
+				$this->crossborder_and_domestic_fields( $order_id, $dhl_label_items, $is_disabled );
```

Comment:
Since its a public function, we should do something like :
```suggestion
				$this->crossborder_and_domestic_fields( $dhl_label_items, $is_disabled, $order_id = 0 );
```

Reply:
ignore this, there is no need to modify the function at all.. we should add the **Germany Only** option within `additional_meta_box_fields` function